### PR TITLE
Added example for web js sdk

### PR DESCRIPTION
**Pull request by MarinaFedy**

### CODE REVIEW

**Review by AlbinaPomogalova — APPROVED** [review]

File: package-lock.json
Comment (serbinadrian) [inline package-lock.json:1]: REMOVE LOCKS via git rm
add to gitignore

Comment (serbinadrian) [reply package-lock.json:1]: + yarn.lock

File: packages/web/examples/calculator/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
+
+In the project directory, you can run:
+
+### `npm start`
```
Comment (serbinadrian) [inline packages/web/examples/calculator/README.md:9]: Wrap these commands into the code block to enable copying

File: packages/web/examples/calculator/public/index.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="SNET-sdk-web wxa,ple calculator"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
+    <!--
```
Comment (serbinadrian) [inline packages/web/examples/calculator/public/index.html:13]: Remove comments

File: packages/web/examples/calculator/src/components/Loader/index.jsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import "./styles.css";
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/Loader/index.jsx:1]: ReactComponent is of ReactElement or ReactNode has to return something;
Consider ReactNode is returning null.

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+            </div>
+            <button 
+                disabled={isDisable || !values.firstValue || !values.secondValue} 
+                onClick={submitAction}
+            >
+                Submit
+            </button>
+            <Error errorMessage={error} />
+            <Loader isLoading={isLoading} />
+            {isComplited && <div className="service-output">
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:91]: completed

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+import { Calculator } from "../../stubs/example_pb_service";
+import Error from "../Error";
+import "./styles.css";
+import Loader from "../Loader";
+
+const ACTIONS = [
+    {value: "add", title: "+"},
+    {value: "sub", title: "-"},
+    {value: "mul", title: "*"},
+    {value: "div", title: ":"},
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:11]: consider /

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    const [values, setValues] = useState({
+        firstValue: '',
+        secondValue: ''
+    });
+    const [response, setResponse] = useState();
+    const [isComplited, setIsComplited] = useState(false);
+    const [error, setError] = useState();
+    const [selectedAction, setSelectedAction] = useState(ACTIONS[0]);
+    const [isLoading, setIsLoading] = useState(false);
+    const [isDisable, setIsDisable] = useState(true);
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:26]: isDisabled
and is[WHAT]Disabled

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+const SUCCESS_CODE = 0;
+
+const ServiceDemo = ({serviceClient}) => {
+    const [values, setValues] = useState({
+        firstValue: '',
+        secondValue: ''
+    });
+    const [response, setResponse] = useState();
+    const [isComplited, setIsComplited] = useState(false);
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:22]: is[WHAT]Completed

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+        if (status !== SUCCESS_CODE) {
+            setError(statusMessage);
+        }
+        setResponse(message.getValue());
+        setIsLoading(false);
+    }
+
+    const submitAction = async () => {
+        try {
+            setIsLoading(true);
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:41]: formatting!

File: packages/web/examples/calculator/src/components/ServiceDemo/index.jsx [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+            setError('Values should be numbers')
+        } else {
+            setIsDisable(false);
+            setError();
+        }
+        setValues({...values, [targetField.name]: targetField.value})
+    }
+
+    return (
+        <>
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceDemo/index.jsx:73]: Consider Fragment

File: packages/web/examples/calculator/src/components/ServiceInfo/index.jsx [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+                <h2>{metadata.display_name}</h2>
+            </div>
+            <div className="groups-container">
+            {metadata.groups.map(group =>
+                <div key={group.group_id} className="service-image-container">
+                    <h3>{group.group_name}:</h3>
+                    {group.pricing.map(price =>
+                        <table className="pricing-info-table" key={price.price_model}>
+                            <tbody>
+                                <tr>
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/ServiceInfo/index.jsx:28]: use meta map rendering

File: packages/web/examples/calculator/src/components/WalletInfo/index.jsx [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+import Loader from "../Loader";
+
+const WalletInfo = () => {
+    const [address, setAddress] = useState('');
+    const [balance, setBalance] = useState('');
+    const [transactionCount, setTransactionCount] = useState('');
+    const [isLoading, setIsLoading] = useState(false);
+
+    const getWalletInfoFromSDK = async () => {
+        setIsLoading(true);
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/WalletInfo/index.jsx:14]: formatting!

File: packages/web/examples/calculator/src/components/WalletInfo/index.jsx [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+        setTransactionCount(transactionCount);
+        setIsLoading(false);
+      }
+
+    return (
+        <div className="wallet-info">
+            <button onClick={getWalletInfoFromSDK}>Get wallet info</button>
+            <Loader isLoading={isLoading} />
+            {address && balance &&
+            <table className="wallet-info-table">
```
Comment (serbinadrian) [inline packages/web/examples/calculator/src/components/WalletInfo/index.jsx:27]: meta map

**Review by serbinadrian — CHANGES_REQUESTED** [review]

**Review by serbinadrian — APPROVED** [review]

**Pull request merged**